tutorials/understanding-feegrant/index.md

```diff
@@ -164,22 +164,22 @@ Before Bob can send `kudos` to Alice, you must set up an allowance for Bob so th
 The `BasicAllowance` is a permission for a grantee to use up fees until the `spend_limit` or `expiration` is reached. Open up a new terminal window and create an allowance with a spend limit of `100000stake` and no expiration date:
 
 ```sh
-$ simd tx feegrant grant $ALICE_KEY $BOB_KEY --from alice --spend-limit 100000stake
+$ simd tx feegrant grant $ALICE $BOB --from alice --spend-limit 100000stake
 ```
 
 View the allowance:
 
 ```sh
-$ simd query feegrant grants $BOB_KEY
+$ simd q feegrant grants $BOB
```

Comment:
```suggestion
$ simd query feegrant grants $BOB
```
That is a correct way of making a query, but for a tutorial, we prefer to have the explicit long form, which makes for easier reading.
The experienced programmer will soon enough be looking for ways to optimise key strokes on their own.